examples/fluids/src/setupdm.c

```diff
@@ -123,7 +156,7 @@ PetscErrorCode VizRefineDM(DM dm, User user, ProblemData *problem, SimpleBC bc,
     if (i + 1 == user->app_ctx->viz_refine) d = 1;
     PetscCall(DMGetVecType(dm, &vec_type));
     PetscCall(DMSetVecType(dm_hierarchy[i + 1], vec_type));
-    PetscCall(SetUpDM(dm_hierarchy[i + 1], problem, d, bc, phys));
+    PetscCall(SetUpDM(dm_hierarchy[i + 1], problem, d, 0, bc, phys));
```

Comment:
We should probably drop a comment here explaining why its ok to have a dissimilar quadrature space on this DM

Reply:
I could also make it have the same quadrature space? I'm up for either, this PR was kinda caught mid-hack to get working code before going back and cleaning up.